pkg/feature/resources.go

```diff
@@ -0,0 +1,34 @@
+package feature
+
+import (
+	"context"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CreateNamespace will create namespace with the given name if it does not exist yet and sets owner, so it will be deleted
+// when a feature is cleaned up.
+func CreateNamespace(namespace string) Action {
```

Comment:
so we have a cluster.CreateNamespace() original as common.CreateNamespace() but we also have a new feature.CreateNamespace() ?

Reply:
Yes, I intentionally left it for the review, as they differ slightly. This one adds owner reference and the other one adds the label. Wrapping the original one into `Action` shouldn't be a big problem, but do we want to have feature owner references? Another way could be to add a reference after the creation but this would result in extra API call.

Reply:
As discussed, we would use a single function with a patch to add `Ownerrefferences`